setup.py

```diff
@@ -28,9 +38,6 @@
         ],
     },
     include_package_data=True,
-    install_requires=[
-        'jsonschema',
-        'six',
-    ],
-    license=about['__license__']
+    install_requires=install_requires if has_enum else (install_requires + ['enum34']),
```

Comment:
Uhh, I'm not sure if that's a good way to handle the dependency. We should probably do this:
https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies

It requires a new enough version of setuptools, which I think is fine for us to depend on.